docs/integrations/sources/google-search-console.md

```diff
@@ -7,6 +7,12 @@ This page contains the setup guide and reference information for the google sear
 
 * Credentials to a Google Service Account \(or Google Service Account with delegated Domain Wide Authority\) or Google User Account
 
+:::note
+
+Google has depreciated [some OAuth workflows](https://developers.google.com/identity/protocols/oauth2/resources/oob-migration) that might impact you. We are in the process of working on a fix at the connector-level. Thank you for your patience!
```

Comment:
Change depreciated > deprecated
Delete "Thank you for your patience!"
Do we have a timeline for the fix? Also, do we have a workaround/alternative for now?
Are other connectors affected by this?

Reply:
No timeline and as far as we know, this only impacts GSC.

Reply:
Okay, if we don't have a timeline for the fix, we shouldn't mention it in docs. How about "Google has deprecated [certain OAuth workflows](https://developers.google.com/identity/protocols/oauth2/resources/oob-migration). Before you use this connector, verify that the deprecated workflows don't impact your setup."

Reply:
We are working on a fix (GL backlog) but we don't have an official timeline as we don't fully understand the scope yet. Note that the OAuth deprecation affects our connector (in other word, customers are impacted as a result of an issue on our side, not their side) so I don't think we can ask them to check their workflows. Do you have some other suggestions on how we can best convey this?

Reply:
Gotcha. In that case, I think we need to give users actionable guidance and ask them to use the alternative method (Service Account, I think?)
How about "Since Google has deprecated certain OAuth workflows, OAuth isn't supported for this connector." And then remove all OAuth-related content from the doc for now

Reply:
Hi Amruta! I think that's fair for now. Can we say "Since Google has deprecated certain OAuth workflows, OAuth isn't supported for this connector at this time.". We are working on a fix so we don't want to give the impression that this is permanent.

Reply:
Yep, sounds good!